lib/avo/resources/resource_manager.rb

```diff
@@ -35,37 +34,30 @@ def build
         #   "FishResource",
         # ]
         def fetch_resources
-          resources = if Avo.configuration.resources.nil?
-            BaseResource.descendants
+          if Avo.configuration.resources.present?
+            load_configured_resources
           else
-            Avo.configuration.resources
+            load_resources_namespace
           end
 
-          resources.map do |resource|
-            if resource.is_a?(Class)
-              resource
-            else
-              resource.to_s.safe_constantize
-            end
+          BaseResource.descendants
```

Comment:
This will return a list of classes that have been loaded, the names of which are necessarily unique.